apps/web/src/pages/integrations/components/NovuSMSProviderModal.tsx

```diff
@@ -0,0 +1,6 @@
+import { ChannelTypeEnum } from '@novu/shared';
+import { NovuProviderBase } from './NovuProviderBase';
+
+export function NovuSMSProviderModal({ onClose }: { onClose: () => void }) {
```

Comment:
```suggestion
export function NovuSmsProviderModal({ onClose }: { onClose: () => void }) {
```
Reminder as one of the first comments in the PR.